test/JDBC/expected/sys-all_sql_modules-vu-prepare.out

```diff
@@ -38,6 +38,13 @@ CREATE PROC sys_all_sql_modules_vu_prepare_p1 AS
+END
+GO
+
```

Comment:
there's no dependent objects

Reply:
Dependency tests are added in `sys-all_sql_modules-dep*` files.